libraries/WebServer/examples/WebServer/README.md

```diff
@@ -28,9 +34,9 @@ It features
 
 Currently, this example supports the following targets.
 
-| Supported Targets | ESP32 | ESP32-S2 | ESP32-C3 |
-| ----------------- | ----- | -------- | -------- |
-|                   | yes   | yes      | yes      |
+| Supported Targets | ESP32 | ESP32-S2 | ESP32-S3 | ESP32-C3 |
+| ----------------- | ----- | -------- | -------- | -------- |
+|                   | yes   | no       | yes      | yes      |
```

Comment:
Please fix S2 and add a C6 to the supported targets. Just tested them and works fine :)
```suggestion
| Supported Targets | ESP32 | ESP32-S2 | ESP32-S3 | ESP32-C3 | ESP32-C6 |
| ----------------- | ----- | -------- | -------- | -------- | -------- |
|                   | yes   | yes      | yes      | yes      | yes      |
```